src/components/figures/figure/figure.ts

```diff
@@ -346,28 +334,31 @@ export class FigureComponent extends Component<Props, SpreadsheetChildEnv> {
     this.dnd.height = figure.height;
 
     const onMouseMove = (ev: MouseEvent) => {
+      const { x: mainViewportX, y: mainViewportY } =
+        this.env.model.getters.getMainViewportCoordinates();
+      const { offsetX: scrollOffsetX, offsetY: scrollOffsetY } =
```

Comment:
is the name scrolloffset necessary ? SheetScrollInfo also container scrollbaroffsetX, they'r a bit to close imo. but nitpick, really

Reply:
I actually had forgotten `SheetScrollInfo` had `offsetScrollbarX` :sweat_smile: 

What I thought was: when I see "offset", I ask myself *"offset of what ?"* If I see the word "scroll" I immediatly know what it is.